app/src/main/kotlin/com/saltedge/authenticator/models/repository/PreferenceRepository.kt

```diff
@@ -150,6 +151,18 @@ object PreferenceRepository : PreferenceRepositoryAbs {
             preferences?.saveValue(KEY_SCREENSHOT_LOCK, value)
         }
 
+    /**
+     * Computed property that read dark mode state from preferences and saves to preferences.
+     *
+     * @return boolean, true if dark mode is enabled
+     * @see saveValue
+     */
+    override var systemDarkMode: Boolean
+        get() = preferences?.getBoolean(KEY_SYSTEM_DARK_MODE, true) ?: true
```

Comment:
instead of hardcoded `true` value it is necessary to check if system dark mode is supported and if current night mode of the application equals to system configuration